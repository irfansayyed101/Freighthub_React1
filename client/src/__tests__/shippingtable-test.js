import React from 'react';
import { shallow } from 'enzyme';
import ShippingTable from '../components/ShipmentTable';

describe('<ShipmentTable /> with props ' , () =>{
    let shipments =  [{
        "id": "S1000",
        "name": "T2-Shirts(Summer 2018) from Shanghai to Hamburg",
        "cargo": [
          {
            "type": "Fabric",
            "description": "1000 Blue T-shirts",
            "volume": "2"
          },
          {
            "type": "Fabric",
            "description": "2000 Green T-shirts",
            "volume": "3"
          }
        ],
        "mode": "sea",
        "type": "FCL",
        "destination": "Saarbr√ºcker Str. 38, 10405 Berlin",
        "origin": "Shanghai Port",
        "services": [
          {
            "type": "customs"
          }
        ],
        "total": "1000",
        "status": "ACTIVE",
        "userId": "U1000"
      }]
    const container = shallow(<ShippingTable shipmement={shipments}/>);
    it('should match the snapshot', () => {
      expect(container.html()).toMatchSnapshot();
    });
});