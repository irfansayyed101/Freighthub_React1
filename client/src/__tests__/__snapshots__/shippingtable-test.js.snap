// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShipmentTable /> with props  should match the snapshot 1`] = `"<table><thead><tr><th>Name</th><th>Mode</th><th>Type</th><th>Destination</th><th>Origin</th><th>Total</th><th>Status</th></tr></thead><tbody><tr><td>T2-Shirts(Summer 2018) from Shanghai to Hamburg</td><td>sea</td><td>FCL</td><td>Saarbr√ºcker Str. 38, 10405 Berlin</td><td>Shanghai Port</td><td>1000</td><td>ACTIVE</td></tr></tbody></table>"`;
